usaspending_api/api_contracts/contracts/v2/reporting/agencies/publish_dates/history.md

```diff
@@ -0,0 +1,106 @@
+
+This endpoint returns an overview of government agencies submission data.
+
++ Parameters
+
+    + `fiscal_year`: 2020 (required, string)
+        The fiscal year.
+        + Default: `All`.
+    + `search`: treasury (optional, string)
+        The agency name to filter on.
+    + `page`: 1 (optional, number)
+        The page of results to return based on the limit.
+        + Default: 1
+    + `limit`: 5 (optional, number)
+        The number of results to include per page.
+        + Default: 10
+    + `order`: `desc` (optional, enum[string])
+        The direction (`asc` or `desc`) that the `sort` field will be sorted in.
+        + Default: `desc`
+        + Members
+            + `asc`
+            + `desc`
+    + `sort`: `current_total_budget_authority_amount` (optional, enum[string])
+        A data field that will be used to sort the response array.
+        + Default: `current_total_budget_authority_amount`
+        + Members
+            + `name`
+            + `current_total_budget_authority_amount`
+            + `period`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + `page_metadata` (required, PageMetaDataObject, fixed-type)
+        + `results` (required, array[SubmissionOverview], fixed-type)
+    + Body
+
+            {
+                "page_metadata": {
+                  "page": 1,
+                  "hasNext": false,
+                  "hasPrevious": false,
+                  "total": 2
+                },
+                "results": [
+                  {
+                    "name": "Department of Health and Human Services",
+                    "abbreviation": "DHHS",
+                    "code": "020",
+                    "current_total_budget_authority_amount": 8361447130497.72,
+                    "periods": [
+                        "period": 2,
+                        "quarter": 1,
+                        "date": "2020-01-20T11:59:21Z",
+                        "certified": true,
+                        "quarterly": false,
+                        "submitted": true
+                    ]
+                  },
+                  {
+                    "name": "Department of Treasury",
+                    "abbreviation": "DOT",
+                    "code": "021",
+                    "current_total_budget_authority_amount": 8361447130497.72,
+                    "periods": [
+                        "period": 2,
+                        "quarter": 1,
+                        "date": "2020-01-20T11:59:21Z",
+                        "certified": false,
+                        "quarterly": false,
+                        "submitted": true
+                    ]
+                  }
+                ]
+            }
+
+# Data Structures
+
+## PageMetaDataObject (object)
++ `page` (required, number)
++ `hasNext` (required, boolean)
++ `hasPrevious` (required, boolean)
++ `total` (required, number)
+
+## Period (object)
++ `period` (required, number),
++ `quarter` (required, number),
```

Comment:
do we need quarter here @ebdabbs @tony-sappe ?

Reply:
Doesn't hurt to include it. Might strike API users as odd if a submission is listed as "quarterly" but only provides the fiscal period